src/main/java/de/uhd/ifi/se/decision/management/jira/persistence/impl/JiraIssueTextPersistenceManager.java

```diff
@@ -45,7 +45,7 @@
  */
 public class JiraIssueTextPersistenceManager extends AbstractPersistenceManagerForSingleLocation {
 	private static final Logger LOGGER = LoggerFactory.getLogger(JiraIssueTextPersistenceManager.class);
-	private static final ActiveObjects ACTIVE_OBJECTS = ComponentGetter.getActiveObjects();
+	static final ActiveObjects ACTIVE_OBJECTS = ComponentGetter.getActiveObjects();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/UHD/cures-condec-jira/pullRequest?prid=4553903)